services/platform/app/features/automations/components/automation-assistant/automation-details-collapse.tsx

```
@@ -0,0 +1,41 @@
+export function AutomationDetailsCollapse({
+  context,
+  title,
+}: AutomationDetailsCollapseProps) {
+  const [isOpen, setIsOpen] = useState(false);
+
+  return (
+    <div className="border-muted mb-2 overflow-hidden rounded-lg border">
+      <button
+        onClick={() => setIsOpen(!isOpen)}
+        className="bg-muted/50 hover:bg-muted flex w-full items-center justify-between px-3 py-2 text-left transition-colors"
+      >
```

Biome (2.3.14)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)